src/Parser/Core/Combatant.js

```diff
@@ -76,7 +76,9 @@ class Combatant extends Entity {
       case SPECS.HOLY_PALADIN.id:
         return 0.12 + this.masteryRating / 26667;
       case SPECS.RESTORATION_SHAMAN.id:
-        return 0.24 + this.masteryRating / 13333.3333333;
+            return 0.24 + this.masteryRating / 13333.3333333;
```

Comment:
Would there be any way you could get your indentation to be consistent with the existing codebase? It's not required to get your changes merged, it's just going to bug me a bit :P